### PR TITLE
Add resource elasticsearch

### PR DESCRIPTION
**Pull request by takaishi**
to spec AWS ElasticsearchService.

```ruby
describe elasticsearch('my-elasticsearch') do
  it { should exist }
  its(:elasticsearch_version) { should eq '2.3' }
  its('elasticsearch_cluster_config.instance_type') { should eq 't2.micro.elasticsearch' }
  it do
    should have_access_policies <<-policy
{
  "version": "2012-10-17",
  "statement": [
    {
      "effect": "allow",
      "principal": "*",
      "action": [
        "es:*"
      ],
      "resource": "arn:aws:es:ap-northeast-1:1234567890:domain/my-elasticsearch/*"
    }
  ]
}
  policy
  end
```

### CODE REVIEW

File: lib/awspec/generator/spec/easticsearch.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+module Awspec::Generator
```
Comment (neil-greenwood) [inline lib/awspec/generator/spec/easticsearch.rb]: typo in filename.

Comment (takaishi) [inline lib/awspec/generator/spec/easticsearch.rb]: thanks, fix it!

**Comment by k1LoW** [comment]
Wow! Great! Thank you!!

**Comment by k1LoW** [comment]
Released as v0.54.0.

**Pull request merged**